src/App.js

```diff
@@ -1,28 +1,10 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/Calculator';
+import './index.scss';
 
+const calculator = new Calculator();
 function App() {
```

Comment:
The App.js component should be class-based 🤓 Please make the necessary changes so that we meet the requirements ✔